maintenance/liftOver_assemblies.py

```diff
@@ -0,0 +1,540 @@
+"""
+Code for liftOver between assemblies
+"""
+
+import sys, os, glob
+import argparse
+import multiprocessing as mp
+import subprocess as sp
+import shlex
+import numpy as np
+import tarfile
+import urllib
+import shutil
+from datetime import date
+import stdpopsim
+import matplotlib.pyplot as plt
+from matplotlib import gridspec
+
+
+def progress_bar(percent, barLen = 50):
+    """
+    prints a progress bar to the screen
+    """
+    sys.stdout.write("\r")
+    progress = ""
+    for i in range(barLen):
+        if i < int(barLen * percent):
+            progress += "="
+        else:
+            progress += " "
+    sys.stdout.write("[ %s ] %.2f%%" % (progress, percent * 100))
+    sys.stdout.flush()
+
+
+def assign_task(tmpFile, task_q, nProcs):
```

Comment:
Seems pretty heavyweight to need our own task queue here - have you tried using concurrent.futures?